sklearn/ensemble/_stacking.py

```diff
@@ -496,6 +505,13 @@ def transform(self, X):
         """
         return self._transform(X)
 
+    def _sk_visual_block_(self):
+        if self.final_estimator is None:
+            final_estimator = LogisticRegression()
+        else:
+            final_estimator = self.final_estimator
```

Comment:
I'm not sure about this one. Would it be fair to say an unfitted estimator here doesn't have an html repr? Or have None as given by the user? Or change the `final_estimator`'s default value to `LogisticRegression`?

Reply:
In the current implementation, an estimator always has an unfitted estimator. In this stacker, if `None` is given by the user, it will become `LogisticRegression`.

Reply:
Then maybe leave a comment about that here and also during validation where `LogisticRegression` replaces `None` so that in the future they don't go out of sync?

Reply:
Ah okay (missed this comment)